package.json

```diff
@@ -39,6 +39,7 @@
   "homepage": "https://github.com/rabix/cwl-ts#readme",
   "dependencies": {
     "ajv": "^6.1.1",
+    "ci": "^1.0.0",
```

Comment:
@kaushik-work why do we need this package?

Reply:
Gah, we don't. I'll remove it.